ddtrace/internal/telemetry/data/metrics.py

```diff
@@ -0,0 +1,58 @@
+import time
+from typing import Dict
+from typing import List
+from typing import Optional
+from typing import Tuple
+
+from ...hostname import get_hostname
+
+
+class Series:
+    """
+    Stores and sends metrics using the Datadog Metrics API: https://docs.datadoghq.com/api/latest/metrics/
```

Comment:
This might not be the best description, since they don't get submitted via the public API.

Might be better to reword as not referencing the docs for the public API?

Or maybe, we can just say producing the same format as what is used for the public API endpoint?

What do you think?

Reply:
Yup that makes sense. On the telemetry intake side we link the API so I went with that convention but you're right it doesn't make sense here